Pear/Controller/ChatViewController.swift

```diff
@@ -62,6 +65,10 @@ class ChatViewController: UIViewController {
         backButton.setImage(UIImage(named: "backArrow"), for: .normal)
         backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
         navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)
+        
+        menuButton.setImage(UIImage(named: "optionicon"), for: .normal)
```

Comment:
talk to mat + noah about creating a file for static declarations of all the `UIImage`'s or something similar.

you can reference the way Volume did it [here](https://github.com/cuappdev/volume-ios/blob/master/Volume/Extensions/Image%2BExtensions.swift)